cmd/skopeo/proxy.go

```diff
@@ -770,23 +896,31 @@ func (buf replyBuf) send(conn *net.UnixConn, err error) error {
 		PipeID:  buf.pipeid,
 	}
 	if err != nil {
+		replyToSerialize.ErrorCode = mapProxyErrorCode(err)
 		replyToSerialize.Error = err.Error()
+
```

Comment:
Non-blocking: Please drop this empty line.

(Also, setting `Error` and `ErrorCode` in the declaration order would be a tiny bit easier to check. I don’t care whether the declaration or this code is adjusted.)